### PR TITLE
Gcc update to 48

### PR DESCRIPTION
**Pull request by vccabral**
## Additions

- piped gulp watch output to developer
- installs gcc to remove optional dependency failures

## Removals

- None

## Changes

- Re-wrote some spacing.

## Review

- @vickumar1981 @kave

### CODE REVIEW

**Comment by kave** [comment]
noice i'll try it later today

File: package.json [hunk 1]
```diff
@@ -60,9 +60,6 @@
     "sinon-chai": "^2.8.0",
     "wcag": "^0.2.2"
   },
-  "optionalDependencies": {
-    "player": "^0.6.1"
```
Comment (anselmbradford) [inline package.json]: This'll break some other code, but I'll push up a PR to take the player out :sob:

Comment (anselmbradford) [inline package.json]: https://github.com/cfpb/cfgov-refresh/pull/1827

**Comment by kave** [comment]
:+1:

File: package.json (hunk 1 above)
Comment (vccabral) [inline package.json]: @vickumar1981 can you solve the merge conflicts?

**Pull request merged**